GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/IPC/MessageListener.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+typedef void (^NewMessageCallback) (NSDictionary *_Nonnull messageInfo);
+
+@interface MessageListener : NSObject
```

Comment:
Use project specific 3-letter prefix for class names

Reply:
Updated to use "VFS".